src/browser-lib/parcel-lib/parcel.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit, OnDestroy, Input, OnChanges, ViewChild, ElementRef } from '@angular/core';
+import { Parcel, ParcelConfig } from 'single-spa';
+
+
+@Component({
+  selector: 'parcel',
+  template: `
+  <div #parcelDiv></div>
+  `
+})
+export class ParcelComponent implements OnInit, OnDestroy, OnChanges {
+  @ViewChild('parcelDiv') parcelDiv: ElementRef;
+  @Input() config: ParcelConfig;
+  @Input() mountParcel: any;
+  @Input() onParcelMount: () => void;
+  @Input() wrapWith = 'div';
+  @Input() customProps: any;
+  @Input() appendTo: any;
+  @Input() handleError = err => console.error(err);
+
+
+  createdDomElement: any;
+  hasError: boolean;
+  unmounted: any;
+  nextThingToDo: Promise<any>;
+  parcel: Parcel;
+
+  constructor() {
+  }
+
+  ngOnInit() {
+    if (!this.config) {
+      throw new Error(`single-spa-angular's Parcel component requires the 'config' prop to either be a parcel config or a loading function that returns a promise. See https://github.com/CanopyTax/single-spa-react`)
```

Comment:
Still references single-spa-react in the link. Change it to single-spa-angular (we'll update the docs later)

```suggestion
      throw new Error(`single-spa-angular's Parcel component requires the 'config' prop to either be a parcel config or a loading function that returns a promise. See https://github.com/CanopyTax/single-spa-angular`)
```